pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[project]
+name = "viscm"
+dynamic = ["version"]
+description = "A colormap tool"
+readme = "README.rst"
+authors = [
+  {name = "Nathaniel J. Smith", email = "njs@pobox.com"},
+  {name = "Stefan van der Walt", email = "stefanv@berkeley.edu"},
+]
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Science/Research",
+  "License :: OSI Approved :: MIT License",
+  "Programming Language :: Python :: 3",
+]
+
+requires-python = "~=3.7"
+dependencies = [
+  "numpy",
+  "matplotlib",
+  "colorspacious",
+  "scipy",
+]
+
+[project.urls]
+repository = "https://github.com/matplotlib/viscm"
+# documentation = "https://viscm.readthedocs.io"
+
+[project.license]
+text = "MIT"
+files = ["LICENSE"]
+
+[project.scripts]
+viscm = "viscm.cli:cli"
+
+
+[build-system]
+requires = ["setuptools", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+zip-safe = false
+packages = {find = {}}
+package-data = {viscm = ["examples/*"]}
+
+
+# [tool.black]
```

Comment:
It'd be fine with me to add black & ruff, but it's also not urgent.

Reply:
I have black, isort, and flake8 ready to go on another branch :)